src/rock/models/feeds/resolver.js

```diff
@@ -24,9 +38,38 @@ export default {
           })
           .map(flatten);
 
-        filterQueries.push(models.Content.find({
+        const showsNews = flatten(channels).includes("news");
+
+        //get user's campus to filter news by
+        let userCampus = (person && showsNews)
+          ? await models.Person.getCampusFromId(person.Id, { cache })
+          : null;
+
+        // TODO: filter other campuses by lookup query here
+        let content = await models.Content.find({
           channel_name: { $or: channels }, offset: skip, limit, status,
-        }, cache));
+        }, cache);
+
+        // logged out only see global news
```

Comment:
This is more a question for @jonhorton, but should a logged out user see *all* news? Or just global news? I go back and forth. @samclaridge thoughts?